app/pages/project/classify.jsx

```diff
@@ -51,8 +41,8 @@ function isPresent(val) {
 
 const classificationQueue = new ClassificationQueue(window.localStorage, apiClient, onClassificationSaved);
 
-auth.listen('change', emptySubjectQueue);
-apiClient.type('subject_sets').listen('add-or-remove', emptySubjectQueue);
+auth.listen('change', classifierActions.emptySubjectQueue);
+apiClient.type('subject_sets').listen('add-or-remove', classifierActions.emptySubjectQueue);
```

Comment:
These handlers should dispatch actions, rather than invoking action creators directly.